README.md

```diff
@@ -23,19 +28,44 @@ This is a JavaScript binding that exposes OpenCV library to the web. This projec
 3. Patch Emscripten & Rebuild.
 
   ```
-  patch -p1 < PATH/TO/patch_emscripten_master.diff
-  ```
-4. Rebuild emscripten
-  ```
+  patch -p1 < PATH/TO/patch_emscripten.diff -d emscripten/master
   ./emsdk install sdk-master-64bit --shallow
   ```
 
-5. Compile OpenCV and generate bindings by executing make.py script.
+4. Compile OpenCV and generate bindings by executing make.py script.
 
   ```
     python make.py
   ```
 
+#### WebAssembly version (experimental)
+
+1. Get the source code (just like above)
+
+2. Install the "incoming" branch of emscripten. You can obtain emscripten by using [Emscripten SDK](https://kripken.github.io/emscripten-site/docs/getting_started/downloads.html).
```

Comment:
Ok, something like that ?
`2. Install the "incoming" branch of emscripten. You can obtain emscripten by using [Emscripten SDK](https://kripken.github.io/emscripten-site/docs/getting_started/downloads.html).
At the time of writing, the current version of Emscripten does not support WebAssembly yet, so you will need to install the "incoming" branch (see as well WebAssembly’s [Developer’s Guide](http://webassembly.org/getting-started/developers-guide/) for a short introduction about WebAssembly).
`